app/src/main/res/drawable/general_item_background_border.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
 <shape xmlns:android="http://schemas.android.com/apk/res/android"
   android:shape="rectangle">
-  <solid android:color="@color/color_def_white" />
+  <solid android:color="@color/component_color_profile_list_activity_profile_view_background_color" />
```

Comment:
Suggest recreating this color independent of activity ... something like
`component_color_general_item_background_border_solid_color`

Reply:
done